pkg/controllers/securitypolicy/securitypolicy_controller.go

```diff
@@ -288,45 +286,34 @@ func (r *SecurityPolicyReconciler) Start(mgr ctrl.Manager) error {
 	return nil
 }
 
-// GarbageCollector collect securitypolicy which has been removed from k8s.
-// cancel is used to break the loop during UT
-func (r *SecurityPolicyReconciler) GarbageCollector(cancel chan bool, timeout time.Duration) {
-	ctx := context.Background()
-	log.Info("garbage collector started")
-	for {
-		select {
-		case <-cancel:
-			return
-		case <-time.After(timeout):
-		}
-		nsxPolicySet := r.Service.ListSecurityPolicyID()
-		if len(nsxPolicySet) == 0 {
-			continue
-		}
-		policyList := &v1alpha1.SecurityPolicyList{}
-		err := r.Client.List(ctx, policyList)
-		if err != nil {
-			log.Error(err, "failed to list SecurityPolicy CR")
-			continue
-		}
+// CollectGarbage collect securitypolicy which has been removed from k8s.
```

Comment:
it's good to add comment:
CollectGarbage implement the interface GarbageCollector method.

Reply:
ok